composer.json

```diff
@@ -26,15 +22,19 @@
         }
     },
     "scripts": {
+        "complete-check": ["@phpstan", "@check-cs", "phpunit"],
+        "phpstan": [
+            "phpstan analyze src --level 5"
+        ],
         "check-cs": [
-            "vendor/bin/php-cs-fixer fix src --diff --dry-run -v --level=symfony",
-            "vendor/bin/php-cs-fixer fix tests --diff --dry-run -v --level=symfony",
-            "vendor/bin/phpcs src tests --colors -p -s --extensions=php --standard=src/ObjectCalisthenics/ruleset.xml"
+            "php-cs-fixer fix src --diff --dry-run -v --rules=@Symfony,-blank_line_after_opening_tag --allow-risky yes",
+            "php-cs-fixer fix tests --diff --dry-run -v --rules=@Symfony,-blank_line_after_opening_tag --allow-risky yes",
+            "phpcs src tests --colors -p -s --extensions=php --standard=src/ObjectCalisthenics/ruleset.xml"
         ],
         "fix-cs": [
-            "vendor/bin/php-cs-fixer fix src -v --level=symfony",
-            "vendor/bin/php-cs-fixer fix tests -v --level=symfony",
-            "vendor/bin/phpcs src tests --colors -p -s --extensions=php --standard=src/ObjectCalisthenics/ruleset.xml"
+            "php-cs-fixer fix src -v --rules=@Symfony,-blank_line_after_opening_tag --allow-risky yes",
+            "php-cs-fixer fix tests -v --rules=@Symfony,-blank_line_after_opening_tag --allow-risky yes",
```

Comment:
Rather add .php_cs file. Also --allow-risky is bad because it can change behavior. I use it only with --dry-run flag for improvement tips.